pkg/tests/tasks/extensions/threescale_wasm_plugin_test.go

```diff
@@ -103,18 +97,32 @@ func TestThreeScaleWasmPlugin(t *testing.T) {
 			// JWT authentication filter is applied only to inbound listeners, so 3scale plugin configured
 			// to use JWT filter metadata always fails on outbound.
 			t.LogStep("Verify that a request from sleep to httpbin returns 403")
-			retry.UntilSuccess(t, func(t test.TestHelper) {
-				oc.Exec(t, pod.MatchingSelector("app=sleep", ns.Foo), "sleep",
-					fmt.Sprintf(`curl http://httpbin:8000/headers -H "Authorization: Bearer %s" -s -o /dev/null -w "%%{http_code}"`, token),
-					assert.OutputContains("403", "Received 403 as expected", "Received unexpected status code"))
-			})
+			sendRequestFromSleepToHttpbin(t, token, "403")
+
+			if env.GetSMCPVersion().Equals(version.SMCP_2_3) {
```

Comment:
The flag APPLY_WASM_PLUGINS_TO_INBOUND_ONLY, will only be set for 2.3 SMCP version of should be for 2.3 +?

Reply:
In this PR I was focused on v2.3. I will submit a follow-up for v2.4, but it will be a little bit more complicated.